web-ng/src/app/components/map/map.component.ts

```diff
@@ -74,4 +73,35 @@ export class MapComponent implements OnInit {
     const navigationExtras: NavigationExtras = { fragment: `f=${featureId}` };
     this.router.navigate([primaryUrl], navigationExtras);
   }
+
+  createMarkerOptions(
+    feature: Feature,
+    focusedFeatureId: string,
+    project: Project
+  ): google.maps.MarkerOptions {
+    // Icon is not yet an input for <map-marker>, this is the only way to change icon for now.
+    // Consider break this down when more inputs are available for <map-marker>.
+    const normalScale = 30;
+    const enlargedScale = 50;
+    const defaultIconColor = 'red';
+    const color =
+      project.layers.get(feature.layerId)!.color || defaultIconColor;
```

Comment:
Doe we want `?` instead of `!` here to avoid failing catastrophically if the data of a single feature is corrupt?